src/mumble/GlobalShortcut_win.cpp

```diff
@@ -637,6 +703,56 @@ QString GlobalShortcutWin::buttonName(const QVariant &v) {
 	}
 #endif
 
+#ifdef USE_XBOXINPUT
+	if (g.s.bEnableXboxInput && xboxinput->isValid() && guid == XboxInputLibrary::quXboxInput) {
+		DWORD idx = (type >> 24) & 0xff;
+		DWORD button = (type & 0x00ffffffff);
+
+		// Translate from our own button index mapping to
+		// the actual Xbox controller button names.
+		// For a description of the mapping, see the state
+		// querying code in GlobalShortcutWin::timeTicked().
+		switch (button) {
+			case 0:
+				return QString::fromLatin1("Xbox%1:Up").arg(idx + 1);
+			case 1:
+				return QString::fromLatin1("Xbox%1:Down").arg(idx + 1);
+			case 2:
+				return QString::fromLatin1("Xbox%1:Left").arg(idx + 1);
+			case 3:
+				return QString::fromLatin1("Xbox%1:Right").arg(idx + 1);
+			case 4:
+				return QString::fromLatin1("Xbox%1:Start").arg(idx + 1);
+			case 5:
+				return QString::fromLatin1("Xbox%1:Back").arg(idx + 1);
+			case 6:
+				return QString::fromLatin1("Xbox%1:LeftThumb").arg(idx + 1);
+			case 7:
+				return QString::fromLatin1("Xbox%1:RightThumb").arg(idx + 1);
+			case 8:
+				return QString::fromLatin1("Xbox%1:LeftShoulder").arg(idx + 1);
+			case 9:
+				return QString::fromLatin1("Xbox%1:RightShoulder").arg(idx + 1);
+			case 10:
+				return QString::fromLatin1("Xbox%1:Guide").arg(idx + 1);
+			case 11:
+				return QString::fromLatin1("Xbox%1:11").arg(idx + 1);
```

Comment:
I take it that one isn't currently triggerable on any controller and doesn't trigger on its own? I guess ``11`` is as good as any other name in that case...will be confusing if any future controller has it though ;) Too bad that there isn't a button -> name API in xinput :(